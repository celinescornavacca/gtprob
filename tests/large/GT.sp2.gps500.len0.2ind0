(((((((((((A201,(A486,A487)),A186),((A217,(A482,A483)),(A205,A206))),A12),((A125,A126),((A215,A216),A81))),(A256,A257)),(((((A47,(A62,(A316,A317))),((A253,A254),A56)),((A289,A290),(A468,A469))),(((A57,A58),(A91,(A141,A142))),A45)),(A8,((A16,(A24,((A398,A399),((A242,(A280,(A448,A449))),(A412,A413))))),((A129,(A183,A184)),((A450,A451),A416)))))),((((A179,(A400,A401)),A61),((A190,A191),A153)),((A243,A244),((A490,A491),A406)))),(((((A198,((A336,A337),A235)),A152),((A432,A433),(A154,A155))),((((((A166,(A230,A231)),A7),((((((A309,A310),((A453,A454),A180)),((A225,(A442,((A472,A473),A447))),(A345,A346))),(((A349,A350),((A372,A373),(A321,A322))),A48)),((A151,(A210,(A274,A275))),(A92,((A434,A435),A295)))),((A307,A308),(A78,(A261,A262))))),((A93,A94),((((A83,A84),((A104,(((A148,(A269,(A344,(A402,(A429,A430))))),A145),A130)),((A220,A221),((A445,A446),A122)))),(A99,A100)),(A333,A334)))),(A2,(((A267,A268),A133),(A21,(((A342,A343),(A218,A219)),A65))))),((A260,(A369,A370)),(A87,A88)))),(((((A40,(((A222,A223),(A110,((A371,(A466,A467)),A313))),A85)),((((A143,(A228,A229)),A139),A118),(A60,(A168,(A258,A259))))),((A359,A360),A140)),(((A233,(A408,A409)),A68),(((A226,(A245,A246)),((A340,A341),A167)),((((A113,((A436,A437),A298)),(A209,(A455,A456))),((A163,(((A330,(A414,A415)),(A214,(A278,A279))),A165)),A132)),((A127,A128),(A380,A381)))))),(A177,A178)))),((((A10,((A355,A356),(A82,((A232,(A291,A292)),A189)))),((A299,A300),((((A265,A266),A176),A80),A76))),((((((((((A196,A197),((A494,A495),A452)),(A236,A237)),((A281,(A311,A312)),(A457,A458))),((((((A273,((A386,A387),A277)),(A374,A375)),(A247,A248)),((A492,A493),(A305,A306))),((((A302,A303),((A417,A418),A335)),(A284,A285)),A66)),(((A382,A383),A173),A51))),(A17,(A498,A499))),((A34,A35),((A29,(((A327,A328),((A410,A411),A238)),((((A318,A319),A208),(A53,(A112,(A249,A250)))),A42))),(A25,((A67,((A363,A364),A101)),A52))))),((((((A19,A20),(A15,(A69,(A207,(A396,A397))))),(A251,A252)),A1),(((A102,A103),A38),A26)),(((A5,((((A394,A395),A329),A77),(A44,(A390,A391)))),(A18,(A367,A368))),((((A263,A264),((A419,A420),(A347,A348))),(A105,A106)),((((A120,A121),A22),((A443,A444),(A272,(A459,A460)))),(A212,A213)))))),(((((((((A314,A315),(A425,A426)),A202),(((A160,(A388,A389)),(A156,A157)),A31)),(A3,((A351,A352),((A107,A108),(A70,A71))))),((((A50,(A484,A485)),A32),(((A270,(A331,A332)),A185),A135)),((A30,(((((A293,A294),(A421,A422)),(A187,A188)),(((A234,(A403,(A480,A481))),A195),A170)),A33)),(A89,(A325,A326))))),((A14,(((A158,A159),A114),A64)),(((A96,A97),(A23,((((A384,A385),(A423,A424)),A90),A86))),((A74,(A123,((((A470,A471),A407),A224),A138))),(((A174,A175),A98),((((A438,A439),A362),A134),((A169,(A282,A283)),A117))))))),(((A27,A28),(((((A41,(A54,A55)),(A463,A464)),(((A227,(A461,A462)),(A376,A377)),A164)),(((A46,(A404,A405)),(A146,A147)),A36)),(((A427,A428),A276),(A476,A477)))),((A296,A297),A211))),(((A4,((A72,A73),(((A353,A354),A200),(A79,((A323,A324),A320))))),(((A239,A240),A144),(((A111,(A338,A339)),(A115,A116)),((A378,A379),A95)))),(((A63,(A287,A288)),((A161,A162),((A392,A393),A49))),((((A357,A358),A199),A13),((A11,((A37,((A488,A489),A465)),((A192,A193),A43))),A9)))))),(((A75,(A181,A182)),((A136,A137),A119)),((A241,((A365,A366),A301)),A39)))),((((A171,A172),(((((((A431,(A474,A475)),A361),A304),A271),A255),(A149,A150)),(A109,(A194,(A478,A479))))),((A124,(((A440,A441),A286),A131)),A0)),(A6,(((A496,A497),(A203,A204)),A59))))),((((((((((B319,(B490,B491)),B294),(B242,B243)),(B365,B366)),(B217,B218)),(((B317,B318),(B322,(B327,B328))),(B381,B382))),((((B133,B134),(((((B151,(B169,B170)),(B498,B499)),(((B148,B149),B130),((B141,B142),B94))),B38),((B86,B87),(B261,(B462,B463))))),(B179,B180)),(((((B64,B65),(((B494,B495),B275),B273)),(B146,(B314,B315))),((B337,B338),B163)),((((((B274,(B446,B447)),((B488,B489),B406)),B105),B92),((B83,B84),B73)),((((((B325,B326),(B335,B336)),B255),B115),B112),B80))))),(B0,((((((((B69,(B269,B270)),(B67,((B329,B330),B143))),((B302,B303),B45)),B13),(B223,(B437,(B492,B493)))),(B102,B103)),(((B47,((B287,B288),B264)),B34),((B106,(B107,(B173,B174))),(B66,(B204,B205))))),((((B176,B177),B33),((B37,(((B396,B397),B234),B91)),((((B251,(B424,B425)),B232),(B158,B159)),(B209,B210)))),((((B373,B374),B46),((((B404,B405),B150),(B285,B286)),B55)),((((B116,(B438,B439)),((B344,B345),B153)),((B8,((((B166,B167),(B472,B473)),B63),B16)),(B281,B282))),((B412,B413),(B444,B445)))))))),(((B240,B241),(((((((B390,B391),(B198,B199)),(B262,B263)),(B202,B203)),(((B10,((B416,B417),B74)),(((((B110,B111),((((B411,(B426,B427)),((B124,(B127,B128)),(B367,B368))),((B182,(B267,B268)),((B181,(B296,B297)),(B394,B395)))),B28)),B14),((B238,B239),(B42,B43))),(((B388,B389),(B196,(B200,B201))),((B58,((B289,B290),B147)),(((B308,B309),(B155,B156)),((((B341,(B400,B401)),B57),B40),(((B224,B225),B101),(((B418,B419),B172),B70)))))))),(((B154,(B246,B247)),B140),B35))),((((((B19,(((B192,(B252,((B363,B364),B259))),B68),(((B283,B284),(B129,((B230,B231),B152))),(B59,B60)))),(((B17,((B464,B465),((B402,B403),((B310,B311),B183)))),(B36,((((B482,B483),(B460,B461)),((B260,(B295,(B323,B324))),B258)),(B117,B118)))),B7)),(((B233,(B253,B254)),(B468,B469)),(B4,B5))),((((B407,B408),B44),((((B164,((B236,B237),B193)),(B144,B145)),((B51,B52),((B97,B98),B54))),(B306,B307))),(((B21,(B93,(B377,B378))),(B332,(B496,B497))),((B139,((B480,B481),B331)),B99)))),(((B11,(B216,((B348,B349),B235))),((((((B278,B279),(B206,B207)),(B12,((B265,B266),(B228,B229)))),(((B320,B321),(B175,((B392,B393),(B291,B292)))),(B385,B386))),(((B18,((B466,B467),B53)),(B20,((B71,(B212,B213)),((B108,B109),(B78,B79))))),(B15,(B211,(B414,B415))))),((B387,(B398,B399)),(B244,B245)))),((((((B135,B136),(B450,B451)),(B442,B443)),(B62,(B90,(B375,B376)))),(((B125,B126),(B81,B82)),(B85,((B276,B277),B226)))),(B56,(B219,B220))))),(((B420,B421),B361),B32))),((((((B190,B191),(B187,B188)),B9),((((B23,B24),((B484,B485),B430)),((B41,(B119,B120)),B27)),((((B184,B185),(B49,B50)),(((B104,((B186,(B456,B457)),(B197,(B227,(B435,B436))))),((B428,B429),B171)),B22)),((B312,B313),((B476,B477),((B160,(B248,B249)),(B168,(B256,B257)))))))),(((B359,B360),(((((B346,((B458,B459),B448)),B157),(B271,B272)),B96),(B29,B30))),B2)),(((B339,B340),(B26,(B221,B222))),((((((B355,B356),B347),(B250,((B486,B487),B449))),(((B39,(B280,(B371,B372))),(B95,(B161,B162))),(B75,B76))),B3),(((((B433,B434),(B189,((B454,B455),(B214,B215)))),B123),((B195,(B316,(B353,B354))),(B342,B343))),(B48,(B208,(B300,B301))))))))),(((B362,(B422,B423)),(((((B194,(B351,B352)),B165),(B409,B410)),(B31,(B137,B138))),B1)),((B452,B453),((((B89,(B440,B441)),((B121,B122),(B113,B114))),(B293,(B333,B334))),(((B379,B380),(B350,(B431,B432))),((B88,(B298,B299)),B72))))))),(((((B470,B471),(B369,B370)),B77),B6),(((B25,(B474,B475)),((B478,B479),((B357,B358),(B383,B384)))),((B131,B132),((B100,(B178,(B304,B305))),B61))))));
