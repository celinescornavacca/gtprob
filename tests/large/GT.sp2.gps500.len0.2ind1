((((((A57,A58),(A105,(A317,(A456,A457)))),(A42,(A55,(A242,A243)))),(((((((((((A172,A173),(A330,A331)),A106),(A419,A420)),((A265,A266),((((A406,A407),(A396,A397)),(A352,(A379,(A386,A387)))),(A308,A309)))),(((((A267,A268),((A69,(A91,A92)),((A179,(A211,(A388,A389))),(A230,A231)))),(A119,A120)),(A107,(A452,A453))),((A46,((A81,(A250,A251)),((A412,A413),(A167,A168)))),((A9,((A33,((A135,A136),(A199,A200))),(A218,(A431,A432)))),(A109,((A277,(A328,A329)),A152)))))),((((((A144,(A365,A366)),A104),(((((A169,A170),A124),(((A216,A217),(A130,A131)),A67)),A27),((A398,A399),((((A367,A368),A116),(A88,(A196,A197))),A84)))),((((A235,A236),(A377,A378)),(A149,(A356,A357))),(A44,(((A164,A165),((A347,(A415,A416)),(A425,A426))),((((A498,A499),A485),A440),A80))))),((A24,A25),(A74,A75))),(((A3,((((A13,A14),((((A103,(A492,A493)),(A249,(A320,A321))),((A292,A293),(A358,A359))),(A38,(((A94,((A362,(A443,A444)),(A190,A191))),A86),(A51,A52))))),(((A240,A241),((A134,(A137,A138)),(A162,A163))),((((A221,(A257,(A342,(A353,(A390,A391))))),((A324,A325),A150)),A47),(((A285,A286),A117),A112)))),((((A232,(A244,A245)),(A438,A439)),(A102,(A125,A126))),((A494,A495),(A476,A477))))),((A228,A229),A222)),(A145,A146)))),((((A132,A133),((A290,A291),A279)),(A66,(A177,(A373,A374)))),((((A338,A339),(A435,A436)),((A192,A193),(A294,(A343,A344)))),((((A19,(((A166,((A400,A401),A254)),((A470,A471),A402)),(((((A350,A351),A238),((A360,A361),A258)),A62),A59))),A8),((((A29,((A155,(A223,A224)),((A480,A481),(A450,A451)))),A22),((A318,A319),A198)),((((A305,A306),(A159,A160)),((A256,(A262,A263)),A239)),(((A39,(A97,(A248,(A417,A418)))),A20),(((A219,A220),A89),(A441,A442)))))),((A54,(A427,A428)),(A18,(((A203,A204),A68),(((A181,((A496,A497),A184)),A90),A48)))))))),A0),((((A87,(A466,A467)),((A260,A261),(((A474,A475),A437),(A180,(A421,A422))))),((((A147,A148),(A101,(A447,(A458,A459)))),(A11,(A72,((A207,A208),(A295,A296))))),((A95,A96),((A382,A383),(A154,(A301,A302)))))),((((A60,A61),((A40,(A111,(A185,((A486,A487),A482)))),((A142,A143),(A64,((A299,A300),A225))))),(A26,(((A113,(A209,A210)),(A178,((A375,A376),A233))),(A129,(A174,A175))))),(((A10,(((A247,(((A488,A489),A255),A252)),(A71,(A118,(A462,A463)))),((A490,A491),(A354,A355)))),((((((A307,(A345,A346)),A158),A141),A100),A78),A12)),((((A30,(((A468,A469),A65),A37)),((((A108,(A213,A214)),(A194,A195)),((A187,(A188,A189)),A99)),A23)),(A70,((A278,(A454,A455)),((A403,(A414,(A448,A449))),A161)))),(A182,A183)))))),(((A15,(A28,((((A380,A381),A259),A43),A31))),(A4,((A483,A484),((((A310,A311),A85),A82),A53)))),(((A334,A335),(A201,A202)),(((((A283,A284),((A433,A434),(A332,A333))),A215),A110),((A21,((((A363,A364),A315),((A121,(A348,A349)),A36)),(A429,A430))),(A153,(A289,(A369,A370))))))))),((A2,((((A336,A337),A157),(A49,A50)),(((((A114,A115),((((A460,A461),(A274,(A275,A276))),((A340,A341),A77)),(A139,A140))),(((A73,(A326,A327)),(A322,A323)),(((A171,(A472,A473)),A83),(((A423,A424),A282),(A408,A409))))),(A34,((A464,A465),A227))),(((A6,A7),(((A45,(A79,((A212,(A226,(A269,A270))),((A205,A206),A151)))),(A122,A123)),(A35,((((A297,A298),A273),A234),(A287,A288))))),(((A93,(A246,((A371,A372),(A478,A479)))),A63),(A404,A405)))))),(A1,((((A280,A281),(A312,A313)),A56),(((((A156,(A410,A411)),((((A264,((A392,A393),A314)),((A394,A395),A316)),A98),A41)),A17),(((A303,A304),((A237,(A384,A385)),A76)),A16)),(A5,(((A127,(((A253,(A271,A272)),A186),A176)),(A128,(A445,A446))),A32))))))),((((((((((((B325,B326),B185),(B126,(B216,B217))),B9),B3),(((B415,B416),(B120,(B143,B144))),((((B165,(B454,B455)),B82),((((B341,B342),B318),B191),B105)),(((((B354,B355),(B214,B215)),(B147,((B395,B396),B294))),(B307,B308)),(B241,(B267,(B358,B359))))))),((B46,B47),((B169,B170),B24))),((B74,B75),(B153,B154))),((((B22,(((((((B462,B463),(B494,B495)),((B331,B332),B328)),(B437,(B472,B473))),(B220,(B230,B231))),(((B50,(B176,B177)),(B419,B420)),(((B312,(B389,B390)),B58),(B76,(B78,(B224,B225)))))),((B404,B405),(B496,B497)))),((B271,B272),B255)),((B14,(((B402,B403),B168),B34)),((B190,(B254,(((B446,B447),B381),(B257,B258)))),(((B259,B260),(B115,B116)),(B232,B233))))),((((((((B261,B262),((B135,B136),((B373,B374),B242))),(((B243,B244),B235),((B124,B125),(B263,B264)))),(((B288,B289),(B128,B129)),((B62,B63),B53))),((B7,(B32,B33)),((B36,(((B468,B469),(B239,B240)),(B484,B485))),((B305,B306),B48)))),((((B164,(B192,(B245,(B335,B336)))),((B488,B489),B178)),((((B204,B205),((((B186,B187),(B100,(B152,(B183,B184)))),B41),((B303,B304),(B151,(B198,(B209,B210)))))),((B269,B270),(B158,(B228,B229)))),((((B323,B324),((B362,(B440,B441)),B353)),(B382,(B460,B461))),B45))),(((((B196,B197),B37),(B54,((B365,(B391,B392)),((B265,B266),B127)))),(((((B383,B384),(B250,B251)),B102),((B319,B320),((B273,((B478,B479),B290)),((B298,(((B456,B457),B330),(B409,B410))),B175)))),(B444,B445))),((((B81,((B202,B203),B161)),((B90,(B113,B114)),B77)),(B146,(B486,B487))),((B59,((B96,B97),B93)),(((B345,((B427,B428),B385)),(B386,(B492,B493))),((B199,((B299,B300),B274)),B139))))))),(((((B480,B481),B370),B38),((B18,B19),((B316,B317),(B159,B160)))),((((B371,B372),(B282,B283)),B13),(((B118,B119),(B387,B388)),(B79,(((B417,B418),(B218,B219)),((B182,(B200,B201)),B108))))))),((B94,B95),((B131,(B280,B281)),((((B398,B399),((((B234,(B498,B499)),B88),((B337,B338),(((B425,B426),B366),B277))),((B301,B302),B212))),((B61,((B452,B453),B397)),(B103,B104))),(((((B295,(B367,(B466,B467))),((B98,(B107,(B309,B310))),(B423,B424))),(((B360,B361),B179),B111)),B2),(B20,B21)))))))),(((((B109,B110),(B86,(B474,B475))),((B5,(((B248,B249),(((((B296,B297),B162),B142),B134),B35)),((B71,((B350,B351),(B252,B253))),B39))),((((B246,B247),B117),(((B91,B92),((((B435,B436),B406),(B348,B349)),(B476,B477))),(B150,(B292,B293)))),(((B431,B432),B211),B133)))),(B4,((B25,(B28,(B121,B122))),B15))),(B207,B208))),(((((((B413,B414),(B482,B483)),(B29,B30)),B1),B0),(((B69,B70),((((B339,B340),B256),B141),(((((B343,B344),B311),B236),B223),B193))),((((((B429,B430),((B171,B172),(B112,(((B421,B422),B189),B188)))),B57),B43),((B314,B315),((B400,B401),B89))),((B16,B17),(B12,(B221,B222)))))),(((((((B157,(B206,((B363,(B450,B451)),B226))),(B163,((B275,B276),B227))),((B407,B408),((B378,B379),B268))),B83),((B333,B334),((B376,B377),B140))),(B6,(B10,(B49,(B173,B174))))),(((B148,B149),((B286,B287),(B99,(B166,B167)))),((B284,B285),(B291,((B464,B465),B352))))))),((((B67,B68),B42),(((B11,(((B55,B56),(B145,(B180,B181))),(((B433,B434),B106),B27))),(B438,B439)),((B64,B65),((B84,B85),(B60,(B278,B279)))))),(((((B87,((B237,B238),B213)),(B155,B156)),B8),(B23,(B44,(((B364,(B470,B471)),((B458,B459),B380)),(B123,(B375,(B442,B443))))))),((((B137,B138),(B72,B73)),(B26,(((B329,(B356,B357)),B130),(B66,(B346,B347))))),((B80,((B194,B195),B101)),(((((B40,(B51,B52)),B31),(((B490,B491),B327),B132)),((B448,B449),((B411,B412),B313))),((B393,B394),((B321,B322),(B368,B369))))))))));
