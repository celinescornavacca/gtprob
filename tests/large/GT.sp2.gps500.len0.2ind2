((((((A63,A64),A47),(A56,((A372,(A422,A423)),(A210,A211)))),((A407,A408),A12)),((A2,((((A185,(A474,A475)),(A460,A461)),(((A37,(((A277,A278),A108),(((A433,A434),A283),A124))),(((A112,A113),(((A431,A432),(A171,(A377,A378))),(A383,A384))),(A385,A386))),((A498,A499),A62))),((A266,A267),(A177,(A335,A336))))),(((((((A30,((((A447,A448),A212),A190),A35)),(((A133,A134),A20),(((((A428,A429),A265),A107),((A100,(A220,A221)),((((A229,A230),(A326,(A472,A473))),(A153,A154)),(A94,A95)))),A0))),(((((((A476,A477),A209),(A482,A483)),A150),((((A126,A127),A114),((A396,A397),A284)),((A338,A339),(((A135,(A207,A208)),(A291,A292)),((A420,A421),(A243,A244)))))),((A65,(A68,((A264,(A418,A419)),(A285,A286)))),(A70,(A147,A148)))),((((A222,A223),(A43,A44)),(A3,A4)),(((A411,A412),(A159,A160)),(A11,((A29,(A342,A343)),(A254,A255))))))),((((A315,A316),((A456,A457),A319)),((A309,A310),(((A162,A163),A99),((((A328,A329),(A275,A276)),(A96,A97)),A73)))),(((A143,((A486,A487),A449)),((((A218,A219),((A102,A103),A92)),((A191,(A226,A227)),A58)),((((A32,(A51,(A86,A87))),((A287,A288),(A375,A376))),A7),((A23,(((A437,(A494,A495)),(A416,A417)),A46)),((A14,A15),(A17,(((A167,(A248,A249)),A128),((A132,(A340,A341)),(A131,(A269,A270)))))))))),(((A74,((A484,A485),A111)),(A478,A479)),((A109,((A480,A481),A168)),((A33,A34),(A180,(A183,A184)))))))),(((A5,((A205,A206),(A241,A242))),(A136,A137)),(((A360,A361),((A213,((A280,(A317,A318)),A271)),(A186,A187))),((A202,(A245,A246)),(A157,A158))))),((((((A75,(A238,A239)),A28),(A80,(A356,A357))),(((((A8,((A26,(((A105,A106),A85),A52)),((A435,A436),A25))),(((A71,A72),((A216,A217),(A346,A347))),A18)),(((((((A450,A451),A101),(((A464,A465),A181),((A260,A261),A55))),(A81,A82)),A6),(((A298,A299),(A45,(A391,(A452,A453)))),((((A350,A351),(A141,A142)),(((A304,A305),(A247,(A324,A325))),A130)),A27))),((((A258,(A466,A467)),(A76,((A262,A263),A119))),(A79,(A492,A493))),A42))),A1),((A193,A194),((A77,A78),(A57,(A91,((A233,A234),A179))))))),(((((A16,((((A370,A371),(A414,A415)),A93),(((A273,A274),(A441,A442)),A121))),((A470,A471),((A364,A365),A240))),(((A59,A60),((A331,A332),((A387,A388),(A443,A444)))),(A36,(A122,(A165,A166))))),((((A488,A489),A235),(((A366,A367),A272),A104)),((A145,A146),(A197,A198)))),((((A344,A345),A330),A192),((((((A468,A469),A413),A402),A337),(A250,A251)),(A496,A497))))),(((A327,(A379,A380)),(((A19,(((A405,A406),A215),A22)),((((A368,A369),(A398,A399)),A38),(A21,(A381,A382)))),(A13,((((((A188,A189),A125),A123),(A462,A463)),A88),(A256,A257))))),((((((A458,A459),(A40,A41)),((A89,A90),(A83,A84))),(((A354,A355),A232),((A169,A170),((((((A348,A349),A182),A117),(A224,A225)),((A66,A67),(A300,A301))),((A403,A404),(A138,(A203,A204))))))),((((A173,A174),((A118,(A296,A297)),(A175,A176))),A31),(((A259,(A389,A390)),((A289,A290),A279)),(A439,A440)))),((((A409,A410),((A308,(A313,A314)),((A155,A156),A110))),(A48,A49)),((A98,((A400,A401),(A358,A359))),((((((A172,((A454,A455),A438)),(A151,A152)),(A430,(A490,A491))),A39),(((((A294,A295),A201),(A228,(A252,A253))),A149),A50)),(((A144,(A362,A363)),(A311,A312)),A69)))))))),((((A9,A10),(((A195,A196),A164),(((((A426,A427),A129),A120),A61),((A53,A54),((A161,((A322,A323),A231)),(A281,A282)))))),((A236,A237),((((A424,A425),(A394,A395)),((A268,(A333,A334)),A178)),(((A320,A321),(A306,A307)),((A352,A353),A214))))),(((A139,A140),((A199,A200),A116)),((((A445,A446),A115),A24),((A373,A374),(((A392,A393),A293),(A302,A303))))))))),((((((B165,B166),B63),B49),((((((B76,(B399,(B434,B435))),(B380,B381)),(((B331,B332),B316),(B110,B111))),((B31,((B126,B127),((B358,B359),(B262,B263)))),(((B364,B365),B98),(B55,B56)))),((B72,B73),(((B149,(B299,B300)),B48),((((B372,B373),(B241,B242)),B102),(B88,(B231,(B317,B318))))))),(((((B216,(B243,B244)),B143),(B42,((((B179,(B181,B182)),B131),B86),(B175,B176)))),(((B227,B228),(B251,B252)),(((B378,B379),((((B454,(B473,B474)),(B266,B267)),(B490,B491)),((B347,B348),B152))),(((B492,B493),B213),(B150,(B264,B265)))))),(((((B124,B125),B41),(((((((((B333,B334),B275),((B432,B433),B357)),B145),(B204,(B428,B429))),(B339,B340)),B93),((B135,B136),(B207,B208))),(B77,B78))),(((B153,B154),(B486,B487)),B74)),((B261,(B440,B441)),((B21,((((((B343,B344),(B183,B184)),B114),B106),B79),((B200,B201),B130))),(B53,(B341,B342)))))))),(((((B390,B391),(((((B384,B385),B32),(((B209,(B452,B453)),B163),B51)),((B26,(B91,(B273,B274))),B19)),B3)),(((((B338,(B369,(B465,B466))),(B279,(B402,B403))),B22),B18),(((((B191,(B278,(B446,B447))),(B211,B212)),B90),(B122,((B480,B481),B148))),(B284,(B362,B363))))),(((((((B189,((B471,B472),B210)),((B156,B157),(B226,(B293,B294)))),((B28,(B29,(B477,(B482,B483)))),((B496,B497),B134))),((B62,(B469,B470)),((B139,B140),(B81,B82)))),(B255,B256)),((((((B158,B159),(B406,B407)),(B115,(((B488,B489),B398),B206))),(B225,(B246,B247))),B40),((B312,B313),B302))),(((B297,B298),((B173,B174),(B351,B352))),(B69,(B455,B456))))),((B13,B14),((((B229,B230),((B367,B368),B162)),B146),B2)))),((((((((B307,B308),(B375,B376)),B20),(((((B360,B361),B235),B108),(B58,(B60,B61))),((((B322,B323),B57),((B137,B138),(B133,(B329,B330)))),B38))),((((((B96,B97),B80),(B54,((B374,(B448,B449)),B314))),((B50,((B327,B328),B107)),((B64,B65),(B33,B34)))),(((((B478,B479),B309),((B170,((B382,B383),B335)),((B217,B218),B141))),B45),B12)),(((B282,B283),B0),((((((B396,B397),B89),(B15,(B95,(B270,(B366,(B420,B421)))))),B4),(((B94,(B151,(B424,B425))),((B355,B356),B92)),(B9,(B52,((B112,B113),(B70,B71)))))),(((B67,(B121,((B438,B439),(B291,B292)))),(B177,B178)),((B186,B187),((B75,((B285,B286),B116)),((B257,(B295,B296)),B161)))))))),((((B416,B417),((((B197,(B370,B371)),B46),B16),B7)),((((B118,B119),(B24,(B39,((B215,(B418,B419)),B44)))),((((B83,(((B237,B238),B132),(B459,B460))),B43),((((B268,B269),B120),B84),B25)),((B280,B281),B194))),(((B223,(B430,B431)),((((B234,(B386,B387)),((B426,B427),((B467,B468),B315))),B180),(B171,B172))),((B319,(B394,B395)),B85)))),(((((((B349,B350),B287),(B253,B254)),((B388,B389),(B408,B409))),B5),(((((B444,B445),(B87,((B310,B311),(B100,B101)))),((((B303,B304),B160),((B276,B277),B103)),B66)),(((B47,(B494,B495)),B35),(B59,(B248,(B324,B325))))),((B192,B193),(B289,B290)))),((((B442,B443),(B224,((B336,B337),B288))),((B232,B233),(B239,B240))),B6)))),((B167,(B377,(B404,B405))),(B326,(B457,B458)))),((((((B117,(B142,(B221,B222))),((B245,(B271,B272)),(B214,(B436,B437)))),((B128,B129),(B144,(B198,B199)))),(((((B414,B415),((B410,B411),B258)),(((B188,(B249,B250)),(B412,B413)),(B205,(B320,B321)))),(B305,B306)),(((B345,B346),(B463,B464)),B17))),(((B23,((B484,B485),B185)),(((B30,((B392,B393),(B109,((B301,(B461,B462)),B190)))),(B400,B401)),B8)),(((B104,B105),(B450,B451)),B1))),((((B195,B196),((B11,((B99,((B155,(B236,(B422,B423))),(B219,B220))),(B259,B260))),B10)),(B123,((B475,B476),(B498,B499)))),(((B168,B169),((((B353,B354),B164),B147),B68)),(((B202,B203),(B36,B37)),B27)))))));
