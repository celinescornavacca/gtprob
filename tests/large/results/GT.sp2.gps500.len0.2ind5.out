-5756.7423329
